pages/linux/zgrep.md

```diff
@@ -0,0 +1,32 @@
+# zgrep
+
+> Grep pattern(s) of text (strings/regex) from files within compressed file (equivalent to grep -Z).
+> Syntax: zgrep [grep_options] pattern file.
```

Comment:
```suggestion
```
This is redundant, as you can see it in the examples. We omit that in tldr pages.

Reply:
removed